lms/static/js/verify_student/views/incourse_reverify_view.js

```diff
@@ -40,13 +40,13 @@
         },
 
         render: function() {
-            var renderedTemplate = _.template($( this.templateId ).html())(
+            var renderedTemplate = HtmlUtils.template($( this.templateId ).html())(
                 {
                     courseKey: this.courseKey,
                     platformName: this.platformName
                 }
             );
-            $( this.el ).html( renderedTemplate );
+            HtmlUtils.setHtml(this.el, renderedTemplate);
```

Comment:
Looks good.  A possible @andy-armstrong nit is that I think he'd prefer these params on their own line with no renderedTemplate variable, and the call to HtmlUtils.template inline here.  At least I think that would be his preference.  But, it is accurate as is.